aten/src/ATen/cpu/vec256/vec256_qint.h

```diff
@@ -1039,7 +1039,7 @@ Vec256<c10::quint8> inline maximum(const Vec256<c10::quint8>& a, const Vec256<c1
   return a.maximum(b);
 }
 
-#else
+#elif !defined(CPU_CAPABILITY_VSX)
```

Comment:
why?

Reply:
it seems redundant. I will remove it and check again. Probably I added it as a compilation for CPU_CAPABILITY_DEFAULT case was failing.